SPECS/policycoreutils/policycoreutils.signatures.json

```diff
@@ -2,13 +2,12 @@
  "Signatures": {
   "policycoreutils-2.9.tar.gz": "c53c344f28007b3c0742bd958751e9b5d2385898adeb8aec6281ae57342f0f7b",
```

Comment:
Please make sure to retarget this PR to the selinux-stabilization branch.

Reply:
Done